doc/reference/orm.rst

```diff
@@ -445,503 +583,173 @@ stored::
 
     nickname = fields.Char(related='user_id.partner_id.name', store=True)
 
-onchange: updating UI on the fly
```

Comment:
why remove all this?

Reply:
Just moved from one place to another ;).